music-journal/src/initFirebase.js

```diff
@@ -2,18 +2,21 @@ import firebase from 'firebase/app';
 import 'firebase/auth';
 
 const config = {
-apiKey: process.env.FIREBASE_API_KEY,
-authDomain: process.env.FIREBASE_AUTH_DOMAIN,
-databaseURL: process.env.FIREBASE_DATABASE_URL,
-projectId: process.env.FIREBASE_PROJECT_ID,
-storageBucket: process.env.FIREBASE_STORAGE_BUCKET,
-messagingSenderId: process.env.FIREBASE_MESSAGING_SENDER_ID
+    apiKey: process.env.FIREBASE_API_KEY,
+    authDomain: process.env.FIREBASE_AUTH_DOMAIN,
+    databaseURL: process.env.FIREBASE_DATABASE_URL,
+    projectId: process.env.FIREBASE_PROJECT_ID,
+    storageBucket: process.env.FIREBASE_STORAGE_BUCKET,
+    messagingSenderId: process.env.FIREBASE_MESSAGING_SENDER_ID
 };
+
 if (!firebase.apps.length) {
-firebase.initializeApp(config);
+    firebase.initializeApp(config);
 }
+
 const auth = firebase.auth();
+
 export {
- auth,
- firebase
+    auth,
+    firebase
 };
```

Comment:
I guess address the missing newlines at end of a few of these files :)

Reply:
Done!